dlfcn_test.go

```diff
@@ -0,0 +1,52 @@
+// SPDX-License-Identifier: Apache-2.0
+// SPDX-FileCopyrightText: 2023 The Ebitengine Authors
+
+//go:build darwin || linux
```

Comment:
If we use `Skipf`, we don't need this restriction. (Rather, it is better to test TestOS for Windows)

Reply:
I believe we still would because it won't compile since there is no `purego.Dlopen` or `purego.RegisterLibFunc` for windows. I think adding stubs is a bad idea since the documentation states that purego.RegisterLibFunc `is not supported on Windows`.

Reply:
I can move TestOS into a different file that can be ran on all platforms.

Reply:
Ah I see, that makes sense.

Reply:
> I can move TestOS into a different file that can be ran on all platforms.

This sounds better.